app/controllers/episode_transcripts_controller.rb

```diff
@@ -0,0 +1,62 @@
+class EpisodeTranscriptsController < ApplicationController
+  before_action :set_episode
+
+  # GET /episodes/1/transcripts
+  def show
+    authorize @episode, :show?
+
+    @episode.assign_attributes(episode_params)
+    @episode.valid?
+  end
+
+  # GET /episodes/1/transcripts/status
+  def status
+    authorize @episode, :show?
+    render :status, layout: false
+  end
+
+  # PATCH/PUT /episodes/1/transcripts
+  def update
+    authorize @episode, :update?
+    @episode.assign_attributes(episode_params)
+
+    respond_to do |format|
+      if @episode.save
+        @episode.copy_media
+        format.html { redirect_to episode_transcripts_path(@episode), notice: t(".notice") }
+      elsif @episode.errors.added?(:base, :media_not_ready)
+
+        # some UI feedback that these files aren't ready
```

Comment:
i wasn't really sure how this would translate over to transcript? would it just be a line saying `@episode.transcript.valid?`

Reply:
looking at `episodes_controller`, i'm inclined to just delete this `contents` code as irrelevant to this controller, but keep this `elsif` branch with the `media_not_ready` error messaging.

Reply:
Yeah, that "media not ready" is a custom error related to publishing.  (You can't publish an episode to the RSS until we've processed its audio files).

I don't think you need that here - it's fine to publish an episode with the transcript not ready yet.